moment/moment-node.d.ts

```diff
@@ -379,11 +379,17 @@ declare module moment {
         meridiem?: (hour: number, minute: number, isLowercase: boolean) => string;
         calendar?: MomentCalendar;
         ordinal?: (num: number) => string;
+        week?: MomentLanguageWeek
```

Comment:
You are right, opened another PR for it! 
https://github.com/DefinitelyTyped/DefinitelyTyped/pull/8471